src/components/SubHeader/EvergreenTaskSubHeader.tsx

```diff
@@ -45,16 +47,20 @@ export const EvergreenTaskSubHeader: React.FC<Props> = ({
 
   const { task } = data ?? {};
   const { logkeeperBuildMetadata } = logkeeperData ?? {};
+  const loadedTask = task ?? logkeeperBuildMetadata?.task;
 
-  if (loading || logkeeperLoading || (!task && !logkeeperBuildMetadata)) {
+  if (loading || logkeeperLoading || !loadedTask) {
     return <Icon glyph="EvergreenLogo" size={24} />;
```

Comment:
Can we show some placeholder text here in case we can't load the task?

Reply:
Good point we can still show the task link like we were doing previously 👍